File: demo/demo/okhttp_example/src/main/java/com/zhm/example/okhttp_example/MainActivity.java
package com.zhm.example.okhttp_example;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        View tvTongbu = findViewById(R.id.tv_tongbu);
        View tvYibu = findViewById(R.id.tv_yibu);

        //同步请求
        tvTongbu.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });

        //异步请求
        tvYibu.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });

    }
}

File: demo/demo/kt_learn191218/src/main/java/com/zhm/example/kt_learn191218/MainActivity.kt
package com.zhm.example.kt_learn191218

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<TextView>(R.id.tv_hello).setOnClickListener(object : View.OnClickListener {
            override fun onClick(v: View?) {
                Toast.makeText(this@MainActivity, "我是提示君~~", Toast.LENGTH_SHORT).show()
            }

        })

        findViewById<TextView>(R.id.tv_jump).setOnClickListener { v ->
            val intent = Intent(this@MainActivity,
                    ListActivity::class.java)
            startActivity(intent)
        }

    }
}
